lib/hanami/model/migrator/logger.rb

```diff
@@ -29,6 +39,22 @@ def _format(hash)
         def initialize(stream)
           super(nil, stream: stream, formatter: Formatter.new)
         end
+
+        # @since <version>
```

Comment:
I'm not sure how we usually feed this information.

Reply:
Usually `# @since x.x.x`

Reply:
Considering the last tag is v1.3.2 I suppose I should increment a patch version and put `since 1.3.3`?

Reply:
Actually I believe it's literally `x.x.x`, and then backfilled at release.